sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -78,13 +79,49 @@ def get_historical_features(
 
         client = _get_bigquery_client()
 
+        entity_df_event_timestamp_col = (
```

Comment:
Ok one more slight nitpick: Do you think we can just pull out all this `entity_df_event_timestamp_col` code into a separate function? I think it will make `get_historical_features()` a bit more readable.